configs/tfm_mbedcrypto_config_profile_medium.h

```diff
@@ -270,7 +270,7 @@
  * Requires: MBEDTLS_PSA_CRYPTO_C
  *
  */
-#define MBEDTLS_PSA_CRYPTO_SPM
+//#define MBEDTLS_PSA_CRYPTO_SPM
```

Comment:
This would be a temporary change until [this change](https://github.com/Mbed-TLS/mbedtls/pull/8164/files#diff-6e9c66b52c2741ccc54517d80ee817de4a80ebf83880131f92bc9d769ddafc01R34) in #8164 goes through, yes?

Reply:
I do wonder if a better temporary fix would be to just have `mtest` turn off `MBEDTLS_PSA_CRYPTO_SPM` for the TFM config

Reply:
Yes, I'm writing a comment there for Yanray

Reply:
Yeah, I'm not sure either now. That's how I originally fixed it, but then thought it would be better if users can build our tfm config out of the box. WDYT?

Reply:
Yes please